apps/juxtaposition-ui/src/services/juxt-web/routes/routeUtils.ts

```diff
@@ -0,0 +1,82 @@
+import type { ParamPack } from '@/types/common/param-pack';
+import type { GetUserDataResponse as AccountGetUserDataResponse } from '@pretendonetwork/grpc/account/get_user_data_rpc';
+import type { Request } from 'express';
+import type { AnyZodObject, z, ZodSchema } from 'zod';
+
+type AnySchema = AnyZodObject | undefined | null;
+
+export type AuthRequest<TReq extends Request = Request> = TReq & {
+	user: AccountGetUserDataResponse;
+	pid: number;
+	tokens: {
+		serviceToken?: string;
+		oauthToken?: string;
+	};
+	paramPackData: null | ParamPack;
+};
+
+export type AuthContext = {
+	user: AccountGetUserDataResponse;
+	pid: number;
+	tokens: {
+		serviceToken?: string;
+		oauthToken?: string;
```

Comment:
We have a type for this already https://github.com/PretendoNetwork/juxtaposition/blob/dev/apps/juxtaposition-ui/src/fetch.ts#L57 - we can move it into types/ and use it everywhere